### PR TITLE
Remove ticks from Refact

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Remove yet more ticks

### CODE REVIEW

File: src/Refact.hs [hunk 1]
```diff
@@ -29,13 +28,10 @@ toRefactSrcSpan' = \case
     GHC.UnhelpfulSpan _ ->
         R.SrcSpan (-1) (-1) (-1) (-1)
 
--- | Don't crash in case ghc gives us a \"fake\" span,
+-- | Dont crash in case ghc gives us a \"fake\" span,
```
Comment (ndmitchell) [inline src/Refact.hs:31]: Do we need to remove 's from `Don't`?

Comment (shayne-fletcher) [reply src/Refact.hs:31]: oops

**Comment by ndmitchell** [comment]
T'h'a'n'k's

**Comment by shayne-fletcher** [comment]
You're welcome
      ^
Plenty more to do 😉

**Pull request merged**